sql/mob_pools.sql

```diff
@@ -115,7 +115,7 @@ INSERT INTO `mob_pools` VALUES (54,'Aerns_Elemental','Aerns_Elemental',100,0x000
 INSERT INTO `mob_pools` VALUES (55,'Aerns_Euvhi','Aerns_Euvhi',109,0x00000E0200000000000000000000000000000000,1,1,11,240,100,0,0,0,1,8,0,0,0,1155,5,0,0,1,0,109);
 INSERT INTO `mob_pools` VALUES (56,'Aerns_Wynav','Aerns_Wynav',193,0x0000120200000000000000000000000000000000,1,1,7,200,100,0,0,0,1,8,1,0,0,1155,0,1,0,1,0,714);
 INSERT INTO `mob_pools` VALUES (57,'Aerns_Xzomit','Aerns_Xzomit',269,0x0000100200000000000000000000000000000000,1,1,7,200,100,0,0,0,1,0,0,0,0,1153,0,0,0,1,0,269);
-INSERT INTO `mob_pools` VALUES (58,'Aestutaur','Aestutaur',240,0x00004F0500000000000000000000000000000000,1,1,0,0,100,0,1,1,0,0,0,0,0,0,0,0,0,1,0,240);
+INSERT INTO `mob_pools` VALUES (58,'Aestutaur','Aestutaur',240,0x00004F0500000000000000000000000000000000,1,1,7,240,100,0,1,1,1,0,0,0,0,0,0,0,0,1,0,240);
```

Comment:
should we make him a monk? not sure wiki is always right, but that is indicated. I'm new to reviewing so if there are nuances to mob_pools jobs then just let me know!